website/docs/quick-start.md

```diff
@@ -7,13 +7,13 @@ slug: /
 
 ## Installation
 
-```
+```bash
 yarn install @buildo/formo
```

Comment:
Sorry for jumping in "all of a sudden": shouldn't this be `yard add ...`?

Reply:
Yeah! Thanks for noticing!